beluga/include/beluga/sensor/data/ndt_cell.hpp

```diff
@@ -33,7 +33,7 @@ namespace beluga {
 
 /// Representation for a cell of a N dimensional NDT cell.
 template <int NDim, typename Scalar = double>
-struct NDTCell {
+struct NdtCell {
```

Comment:
@nahueespinosa meta: why the name change? NDT is an acronym. If we were to spell it out in full it would read `NormalDistributionTransformCell`, emphasis on capital letters. Same elsewhere.

Reply:
We say `NDTSensorModel` and `NDTCell`, but we also say `NdtAmclNode`. What should we do with the latter? `NDTAMCLNode`?

Reply:
Hmm, mixed case acronyms don't stand out as much when parsing code. Alas, `NDTAMCLNode` _is_ almost excessive :thinking:

I like neither. Short term, let's just be consistent. Too many acronyms for a single class may suggest the class is bundling too much, but that's out of scope.